ui/src/main/webapp/home/index.js

```diff
@@ -113,31 +66,36 @@ const LdapTileView = (props) => {
     baseUserDn,
     onDeleteConfig,
     ldapUseCase,
-    attributeMappings
+    attributeMappings,
+    submitting
   } = props
 
   return (
-    <Paper className={styles.config}>
-      <div className={styles.tileTitle}>{ldapUseCase === 'authentication' ? 'LDAP Authentication Source' : 'LDAP Attribute Store'}</div>
-      <ConfigField fieldName='Hostname' value={hostName} />
-      <ConfigField fieldName='Port' value={port} />
-      <ConfigField fieldName='Encryption Method' value={encryptionMethod} />
-      <ConfigField fieldName='Bind User' value={bindUser} />
-      <ConfigField fieldName='Bind User Password' value='******' />
-      <ConfigField fieldName='UserName Attribute' value={userNameAttribute} />
-      <ConfigField fieldName='Base Group DN' value={baseGroupDn} />
-      <ConfigField fieldName='Base User DN' value={baseUserDn} />
-      { ldapUseCase !== 'authentication'
-        ? (
-          <MapDisplay label='Attribute Mappings'
-            mapping={attributeMappings} />
-      ) : null }
-      <RaisedButton style={{marginTop: 20}} label='Delete' secondary onClick={onDeleteConfig} />
+    <Paper className={styles.outerSpinner}>
+      <Spinner submitting={submitting}>
+        <div className={styles.innerSpinner}>
```

Comment:
I don't think this is going to be possible when it comes to editing these configurations. We would have to send detailed information about the config such as the component type to render for the field, values possible to select from, add conditional logic such as "when I select this field, don't show this other, etc.", at which point we're writing frontend code on the backend. This was one of the lessons we learned on a previous iteration that we really disliked.